fastlane/metadata/default/description.txt

```diff
@@ -19,4 +19,8 @@ WooCommerce is the world’s most customizable open-source eCommerce platform. W
 
 Requirements: WooCommerce v3.5+.
 
+View the Terms of Service at https://wordpress.com/tos/
```

Comment:
I was surprised the ToS didn't have a WooCommerce specific URL. 

TIL there's a reason for that : https://woocommerce.com/posts/terms-of-service-update/?quid=dc1998d9a92ed20aeb4c3b1ec5eff158